packages/components/src/popover/index.js

```diff
@@ -42,6 +42,24 @@ const FocusManaged = withConstrainedTabbing(
  */
 const SLOT_NAME = 'Popover';
 
+function offsetIframe( rect, ownerDocument ) {
+	const { defaultView } = ownerDocument;
+	const { frameElement } = defaultView;
```

Comment:
```suggestion
	const { defaultView: { frameElement } } = ownerDocument;
```

Reply:
Both constants are needed.

Reply:
Both are still usable in my suggestion, but I'm just nitpicking here, so it really doesn't matter. :stuck_out_tongue: